src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -314,16 +315,29 @@ impl Service<Vec<Event>> for CloudwatchLogsSvc {
 
     fn call(&mut self, req: Vec<Event>) -> Self::Future {
         if self.token_rx.is_none() {
-            let events = req
+            // We need to take into account the flight time from here to aws servers,
+            // so we are reducing timestamp acceptable range from both sides
+            // for this amount.
+            let buffer_time = Duration::minutes(1);
```

Comment:
No, just a speculation and defensive programming on my side since I'm not sure how strictly does aws check for this.  

> two hours is the buffer 

Now I see, yes this is not needed, I'll remove it.